benchmark/micro/json/RealNest/5_list_filter_on_unnested_structure.benchmark

```diff
@@ -0,0 +1,21 @@
+# name: benchmark/micro/json/RealNest/5_list_filter_on_unnested_structure.benchmark
+# description: list_filter on unnested_muon structure
+# group: [RealNest]
+
+name list_filter_on_unnested_structure
+group real_nest
+
+require json
+
+load
+CREATE TABLE Run2012B_SingleMu AS SELECT * FROM read_json('duckdb_benchmark_data/hep-adl-ethz-Run2012B_SingleMu/data.jsonl');
+
+run
+SELECT list_filter(
+        [pt, eta, phi, mass, pfRelIso03_all, pfRelIso04_all, dxy, dxyErr, jetIdx, genPartIdx],
```

Comment:
Are there lists in the data set? It would be nice to benchmark `list_filter` without the overhead of first creating the list like here. Nonetheless, we should keep this benchmark 👍

Reply:
There are multiple lists in this data set: Muon, Electron, Tau, Photon, Jet.
And yes, it is interesting to try to benchmark them without creating the list, I'll see if I could do that)

Reply:
Great. :)